### PR TITLE
Require is deprecated on 0.4

### PR DESCRIPTION
**Pull request by abelsiqueira**
`require` is deprecated on 0.4

### CODE REVIEW

File: src/CUTEst.jl [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 module CUTEst
 
-require(Pkg.dir("MathProgBase","src","NLP","NLP.jl"))
+include(Pkg.dir("MathProgBase","src","NLP","NLP.jl"))
```
Comment (dpo) [inline src/CUTEst.jl]: Doesn't seem to work in 0.3:
```JULIA
julia> using CUTEst
ERROR: NLP not found
```

**Comment by abelsiqueira** [comment]
Ah, I still have the old NLP so it didn't break. Travis was also downloading it. I will try to change this another way.

**Comment by dpo** [comment]
Good point. I forgot to change `.travis.yml`!

**Comment by abelsiqueira** [comment]
I hope I've fixed it by using `push!(LOAD_PATH,...)`. This allows `using NLP` without warnings.
Also removed NLP from travis.

**Comment by dpo** [comment]
Works for me. Thanks.

**Pull request merged**